patroni/api.py

```diff
@@ -180,6 +180,80 @@ def do_GET_config(self):
+                       " transaction log, 0 if this node is not the leader.")
+        metrics.append("# TYPE patroni_xlog_location counter")
+        metrics.append("patroni_xlog_location {0}".format(postgres.get('xlog', {}).get('location', 0)))
+
+        metrics.append("# HELP patroni_standby_leader Value is 1 if this node is the standby_leader, 0 otherwise.")
+        metrics.append("# TYPE patroni_standby_leader gauge")
+        metrics.append("patroni_standby_leader {0}".format(int(postgres['role'] == 'standby_leader')))
+
+        metrics.append("# HELP patroni_replica Value is 1 if this node is a replica, 0 otherwise.")
+        metrics.append("# TYPE patroni_replica gauge")
+        metrics.append("patroni_replica {0}".format(int(postgres['role'] == 'replica')))
+
+        metrics.append("# HELP patroni_xlog_received_location Current location of the received"
+                       " Postgres transaction log, 0 if this node is not a replica.")
+        metrics.append("# TYPE patroni_xlog_received_location counter")
+        metrics.append("patroni_xlog_received_location {0}".format(
+                        postgres.get('xlog', {}).get('received_location', 0)))
+
+        metrics.append("# HELP patroni_xlog_replayed_location Current location of the replayed"
+                       " Postgres transaction log, 0 if this node is not a replica.")
+        metrics.append("# TYPE patroni_xlog_replayed_location counter")
+        metrics.append("patroni_xlog_replayed_location {0}".format(
+                        postgres.get('xlog', {}).get('replayed_location', 0)))
+
+        metrics.append("# HELP patroni_xlog_replayed_timestamp Current timestamp of the replayed"
+                       " Postgres transaction log, 0 if null.")
+        metrics.append("# TYPE patroni_xlog_replayed_timestamp counter")
+        xlog_replayed_timestamp = postgres.get('xlog', {}).get('replayed_timestamp', 0)
+        metrics.append("patroni_xlog_replayed_timestamp {0}".format(
+                        xlog_replayed_timestamp if xlog_replayed_timestamp is not None else 0))
+
+        metrics.append("# HELP patroni_xlog_paused Value is 1 if the Postgres xlog is paused, 0 otherwise.")
+        metrics.append("# TYPE patroni_xlog_paused gauge")
+        metrics.append("patroni_xlog_paused {0}".format(
+                        int(postgres.get('xlog', {}).get('paused', False) is True)))
+
+        metrics.append("# HELP patroni_postgres_server_version Version of Postgres.")
+        metrics.append("# TYPE patroni_postgres_server_version gauge")
+        metrics.append("patroni_postgres_server_version {0}".format(postgres['server_version']))
+
+        metrics.append("# HELP patroni_cluster_unlocked Value is 1 if the cluster is unlocked, 0 if locked.")
+        metrics.append("# TYPE patroni_cluster_unlocked gauge")
+        metrics.append("patroni_cluster_unlocked {0}".format(int(postgres['cluster_unlocked'] == 'true')))
```

Comment:
The cluster_unlocked is boolean.
```suggestion
        metrics.append("patroni_cluster_unlocked {0}".format(int(postgres['cluster_unlocked'])))
```